src/vaults/SyncRequests.sol

```diff
@@ -315,7 +317,13 @@ contract SyncRequests is BaseInvestmentManager, ISyncRequests {
         balanceSheet.issue(poolId, scId, receiver, priceData.poolPerShare, shares);
 
         balanceSheet.deposit(
-            poolId, scId, vaultDetails.asset, vaultDetails.tokenId, escrow, depositAssetAmount, priceData.poolPerAsset
+            poolId,
+            scId,
+            vaultDetails.asset,
+            vaultDetails.tokenId,
+            address(poolEscrowProvider.poolEscrow(poolId.raw())),
```

Comment:
Nit: if you put `address escrow =  address(poolEscrowProvider.poolEscrow(poolId.raw()))` on the line above this, I believe the rest of the call would fit on 1 line.